recipes/embag/all/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+cmake_minimum_required(VERSION 3.15)
```

Comment:
I guess this file is replacing the native Bazel build system.
In principle, recipes should try to use the native build system as much as possible, because replacing it might have unintended consequences. Did you try to use the ``bazel`` package that exists in ConanCenter as ``tool_requires``?

Reply:
@memsharded Could this recipe be merged as-is and the Bazel-based build added in a future PR, perhaps?

Reply:
@valgur Thank you for your feedback, it's important for improvement of Bazel support on Conan client! Could you please open an issue to https://github.com/conan-io/conan/issues with your current comments? Bazel support is still not documented in docs.conan.io, and any kind of detail would help us to integrate it even better.

Reply:
Done: https://github.com/conan-io/conan/issues/14519